### DISCUSSION TITLE
Ignore column on export

### DISCUSSION THREAD

**Discussion by OrakFR** [Power-Components/livewire-powergrid#1518] [Q&A] [answered]
First of all, I'd like to express my appreciation for livewire-powergrid and to thank you @luanfreitasdev and the entire community for your dedication to developing and maintaining it.

I have a question regarding the export functionality:
Is there a way to exclude specific columns from being exported to Excel or CSV in PowerGrid, even if they are visible on the screen? While I know that hiding the columns can accomplish this, I'm looking for a method to selectively force control the visibility of columns specifically for export purposes.

Any assistance would be appreciated.

**Comment by dansysanalyst** [comment, marked as answer]
Hi @OrakFR,

You can use [visibleInExport](https://livewire-powergrid.com/table/include-columns.html#visibleinexport).

For example:

```php
//MyPowergridTable.php

 public function columns(): array
 {
    return [
        Column::make('ID', 'id')
            ->searchable()
            ->visibleInExport(false) //Hide in export
            ->sortable(),

        Column::make('Name', 'name')
            ->searchable()
            ->hidden()
            ->visibleInExport(true)  //Show in export
            ->sortable(),

        Column::make('Created At', 'created_at_formatted') // By default, shows in export.
    ];
  }
```

**Reply by OrakFR** [reply]
Thank you @dansysanalyst! I missed it in the documentation...
